Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -1127,37 +1181,10 @@ static ErrorOr<void> parse_header(AK::SeekableStream& stream, JPEGLoadingContext
         case JPEG_EOI:
             dbgln_if(JPEG_DEBUG, "{}: Unexpected marker {:x}!", TRY(stream.tell()), marker);
             return Error::from_string_literal("Unexpected marker");
-        case JPEG_APPN0:
-        case JPEG_APPN1:
-        case JPEG_APPN2:
-        case JPEG_APPN3:
-        case JPEG_APPN4:
-        case JPEG_APPN5:
-        case JPEG_APPN6:
-        case JPEG_APPN7:
```

Comment:
not sure this commit is overall a win. it's more code, and not clearly better imho. (it splits the marker switch into several switch statements, and generally doesn't seem like it helps much.)

That's admittedly fairly subjective!

Reply:
I updated the commit message